cmd/csv-merger/csv-merger.go

```diff
@@ -75,217 +108,227 @@ var (
 	ovsCsv         = flag.String("ovs-csv", "", "OVS CSV filename")
 	cinderCsv      = flag.String("cinder-csv", "", "Cinder CSV filename")
 	csvOverrides   = flag.String("csv-overrides", "", "CSV like string with punctual changes that will be recursively applied (if possible)")
+	importEnvFiles = flag.String("import-env-files", "", "Comma separated list of file names to read default operator ENVs from. Used for inter-bundle ENV merging.")
+	exportEnvFile  = flag.String("export-env-file", "", "Name the external file to write operator ENVs to. Used for inter-bundle ENV merging.")
+	almExamples    = flag.Bool("alm-examples", false, "Merge almExamples into the CSV")
```

Comment:
According to the way we call `csv-merger` in both storage and custom bundle this is always `false`: I'm getting it right? This means that inspecting the generated `clusterversion` I should see `alm-examples: []`.

Reply:
Ah ok, I see:

```
  annotations:
    alm-examples: ""
```

so the examples are shown only in the top-level bundle (`openstack-operator`)